visualization/dashboard/src/Localization/ru-RU.json

```diff
@@ -0,0 +1,109 @@
+{
+    "selectPoint": "Выберите точку, чтобы просмотреть объяснение ее локальной важности.",
+    "defaultClassNames": "Класс {0}",
+    "defaultFeatureNames": "Функция {0}",
+    "absoluteAverage": "Среднее абсолютного значения",
+    "predictedClass": "Прогнозируемый класс",
+    "dataExploration": "Исследование данных",
+    "globalImportance": "Глобальный уровень важности",
+    "explanationExploration": "Изучение объяснения",
+    "summaryImportance": "Суммарная важность",
+    "featureImportance": "Важность признака",
+    "perturbationExploration": "Исследование изменений",
+    "localFeatureImportance": "Локальная важность признака",
+    "ice": "ICE",
+    "clearSelection": "Очистить выделение",
+    "feature": "Компонент:",
+    "intercept": "Отсекаемый отрезок",
+    "ExplanationScatter": {
+      "dataLabel": "Данные: {0}",
+      "importanceLabel": "Важность: {0}",
+      "predictedY": "PredictedY",
```

Comment:
it looks like this is not getting translated on any of them?

Reply:
interesting, it may be due to the lack of a space? I can look at that later

Reply:
you're probably right, it seems to be a common theme for all of the languages